src/client/lib/flux/flux.js

```diff
@@ -1,17 +1,22 @@
 import EventEmitter from 'eventemitter3';
+import invariant from 'invariant';
 
 // That's all we need for hot reloadable atomic Flux.
 // Thank you Samuel - https://github.com/vacuumlabs/vlux
 
 export default class Flux extends EventEmitter {
 
-  constructor(store, state) {
+  constructor(store, state?) {
```

Comment:
In React Native and generally, in few use cases, there's no `initialState` given and it cannot be considered an anti-pattern, so I am making it optional.